src/pages/index.js

```diff
@@ -26,29 +28,80 @@ const profilePopup = new PopupWithForm(
 profilePopup.setEventListeners();
 const newCardPopup = new PopupWithForm("#newcards", addNewCardSubmit);
 newCardPopup.setEventListeners();
+const EditPfp = new PopupWithForm("#pfpEdit", editPfpSubmit);
+EditPfp.setEventListeners();
+const confirmPopup = new PopupWithConfirm("#confirm");
+confirmPopup.setEventListeners();
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "8d3dab07-3d12-4401-9299-225a306c1793",
+    "Content-Type": "application/json",
+  },
+});
 
-function handleCardImageClick(name, link) {
-  imagePopup.open(name, link);
+//initalization
+function createCard(cardData) {
+  const card = new Card(
+    cardData,
+    "#jscard-template",
+    handleCardImageClick,
+    handleDeleteCard,
+    handleLikeCard
+  );
+  const cardElement = card.getView();
+  cardSection.addItem(cardElement);
 }
+
+function cardListData() {
+  return api
+    .getInitialCards()
+    .then((res) => {
+      console.log("Fetched Cards:", res);
+      return res.map((card) => ({
+        name: card.name,
+        link: card.link,
+        _id: card._id,
+        isLiked: card.isLiked,
+      }));
+    })
+    .catch((err) => {
+      console.error(err);
+      return [];
+    });
+}
+
+cardListData()
+  .then((data) => {
+    const cardElement = data.map((cardData) => createCard(cardData));
+    cardSection.renderItems(cardElement);
```

Comment:
You need to use `renderItems` for rendering the array of cards. 

Now `renderItems` doesn't render them here because it doesn't accept arguemets in `Section` https://snipboard.io/FjHpVY.jpg . Please, make it accept the array